tests/Unit/Expressions/ForClasses/DummyClasses/Banana.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Arkitect\Tests\Unit\Expressions\ForClasses\DummyClasses;
+
+class Banana implements FruitInterface
```

Comment:
if possibile I would avoid creating test classes here. You could either:

1.  create fixtures directory like we did in the E2E section of the tests
2.  rely on vfsstream and create the classes only in memory. This also means you'll have to create and register a custom autoloader. (that would be my preferred choice but if it is too much work no 1. is fine

Reply:
I moved the dummy classes declaration to inside the class test itself, this avoids creating the dummy classes files, keeps the whole test self-contained and avoids the complexity of using a vfs.

Let me know if this is ok.